backend/src/lib.rs

```diff
@@ -62,6 +62,9 @@ pub struct Halo2MockBackend;
 #[cfg(feature = "halo2")]
 pub struct Halo2AggregationBackend;
 
+#[cfg(feature = "nova")]
+pub struct NovaBackend;
```

Comment:
Notice that after 417 merge, if adding `asm` raw parsed data then it will affect other backend, i.e. Halo2 as well, since now Nova still need to parse information from powdr-asm meta. Will rebase later once we figure out a better way to deal with meta info from PIL directly.